anime_downloader/sites/animesvostfr.py

```diff
@@ -0,0 +1,262 @@
+            # 'photo': ''
+            #
+            #
+            urls_server = {} 
+            for serv in self.SERVERS:
+                urls_server[serv] = helpers.get(self.link_stream_url, params={'server' : serv, 'filmId' : filmid}).text.strip()
+               
+            logger.debug( "_get_sources urls: {}".format(urls_server) )
+            
+            sources = []
+  
+            for serv in self.SERVERS:
+                if urls_server[serv] and (urls_server[serv] > '') and (urls_server[serv].find('No m3u8Id') < 0):
+                
+                    soup = None
+                    try:
+                        soup = helpers.soupify(helpers.get(urls_server[serv]))
+                    except HTTPError as err:
+                        logger.debug("_get_sources HTTP Err: %s", str(err))
+                    
+                    if soup:
+                        #logger.debug( "soup: {}".format(soup) )
+                        iframe = soup.select_one("iframe")
+                        if iframe:
+                            url = iframe['src'].strip()
+                            if url and (url > ''):
+                                logger.debug( "%s -> %s", urls_server[serv], url )
+                                urls_server[serv] = url 
+                            
+                    #video = soup.select_one("video") 
+                    #if video:
+                    #    logger.debug( "_get_sources urls: {}".format(video) )
+                        
+                    
+                    #if serv == "rapidvideo":
+                    #    sources.append(('no_extractor', urls_server[serv]))
+                    if 'youtubedownloader' in urls_server[serv]:
+                        sources.append({'extractor':'gcloud', 'server':'youtubedownloader', 'url':urls_server[serv], 'version':'subbed'})
+                        
+                    elif 'gcloud' in urls_server[serv]:
+                        sources.append({'extractor':'gcloud', 'server':'gcloud', 'url':urls_server[serv], 'version':'subbed'})
+                    
+                    elif 'feurl' in urls_server[serv]:
+                        sources.append({'extractor':'gcloud', 'server':'feurl', 'url':urls_server[serv], 'version':'subbed'})
+                            
+                    elif 'fembed' in urls_server[serv]:
+                        sources.append({'extractor':'gcloud', 'server':'fembed', 'url':urls_server[serv], 'version':'subbed'})
+                    
+                    elif 'hydrax' in urls_server[serv]:
+                        sources.append({'extractor':'hydrax', 'server':'hydrax', 'url':urls_server[serv], 'version':'subbed'})
+                        
+                    else:
+                        logger.warn( "_get_sources url: %s not supported", urls_server[serv])
+               
+            # TODO: put it into config.py   
+            if not self.sitename in Config['siteconfig']:         
```

Comment:
What is this? Don't modify `Config` like this. What do you intend to achieve with this?

Reply:
it was temporary for make test with sort_sources function , it is no longer relevant